src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1263,8 +1355,26 @@ module Mutations = struct
   let unlock_wallet =
     io_field "unlockWallet"
       ~doc:"Allow transactions to be sent from the unlocked account"
-      ~typ:(non_null Types.Payload.unlock_wallet)
-      ~args:Arg.[arg "input" ~typ:(non_null Types.Input.unlock_wallet)]
+      ~deprecated:(Deprecated (Some "use unlockAccount instead"))
+      ~typ:(non_null Types.Payload.unlock_account)
+      ~args:Arg.[arg "input" ~typ:(non_null Types.Input.unlock_account)]
+      ~resolve:(fun {ctx= t; _} () (password, pk) ->
+        let password = lazy (return (Bytes.of_string password)) in
+        match%map
+          Coda_lib.wallets t |> Secrets.Wallets.unlock ~needle:pk ~password
+        with
+        | Error `Not_found ->
+            Error "Could not find owned account associated with provided key"
+        | Error `Bad_password ->
+            Error "Wrong password provided"
+        | Ok () ->
+            Ok pk )
+
+  let unlock_account =
```

Comment:
Nit: The resolve functions for unlock_account/unlock_wallet, lock_wallet/lock_account, delete_wallet/delete_account and reload_wallet/reload_account seem identical. Can you dedup them?